tests/check_framework/test_security.py

```diff
@@ -394,6 +394,10 @@ def test_missing_secret_key(self):
     def test_none_secret_key(self):
         self.assertEqual(base.check_secret_key(None), [base.W009])
 
+    @override_settings(SECRET_KEY='django-insecure-' + ('abcdefghijklmnopqrstuvwx' * 2))
```

Comment:
We can use the constant here and the prefix also contains enough unique characters, so we can simplify:
```suggestion
    @override_settings(SECRET_KEY=base.SECRET_KEY_INSECURE_PREFIX + 'x' * 34)
```